go.mod

```diff
@@ -2,128 +2,123 @@ module github.com/kanopy-platform/argoslower
 
 go 1.21
```

Comment:
Lets tackle an upgrade to go 1.23 as well as the Security issues in a separate PR.